dev/com.ibm.ws.wsat.recovery_fat.multi.9.ee10/fat/src/suite/FATSuite.java

```diff
@@ -44,5 +44,5 @@ public class FATSuite extends TxTestContainerSuite {
 
     @ClassRule
     public static RepeatTests r = RepeatTests.withoutModification()
-                    .andWith(FeatureReplacementAction.EE10_FEATURES());
+                    .andWith(FeatureReplacementAction.EE10_FEATURES().fullFATOnly());
```

Comment:
is this FATSuite supposed to be for EE 10?  If so why does it do withoutModification to begin with?